signals.mss

```diff
@@ -240,6 +240,33 @@ Format details:
     marker-allow-overlap: true;
   }
 
+  /*********************************************/
+  /* DE wrong road signal Zs 6(DB) /Zs 7(DR)  */
+  /*********************************************/
+  /* sign */
+  [zoom>=17]["wrong_road"="DE-ESO:zs6"]["wrong_road_form"="sign"] {
+    marker-file: url('symbols/de/zs6-sign.svg');
+    marker-width: 7;
+    marker-height: 16;
+    marker-allow-overlap: true;
+  }
+
+  /* light DB version */
+  [zoom>=17]["wrong_road"="DE-ESO:db:zs6"]["wrong_road_form"="light"] {
+    marker-file: url('symbols/de/zs6-db-light');
+    marker-width: 7;
+    marker-height: 16;
```

Comment:
Ratio of width and height does not match the ratio of the SVG file.

Reply:
Damn, sorry, should be solved now. Thanks!